src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -240,7 +249,9 @@ export default {
 		},
 
 		isLinkComponent() {
-			return this.hasUrl ? 'a' : 'div'
+			return this.hasUrl
+				? (this.to ? RouterLink : 'a')
+				: 'div'
```

Comment:
Can we address this in follow-up, maybe? or `<a role="button">` would suffice?
```suggestion
				: (this.popoverEmpty ? 'div' : 'button')
```

Reply:
Yes, let's do it in a follow up. It is not a part of the original issue. TBH I even don't remember, where we use use bubble with popover...